Makefile

```diff
@@ -145,3 +145,13 @@ check-golangci-lint-install:
 ifeq ("$(shell command -v golangci-lint)", "")
 	$(error golangci-lint was not found.  Please install it using the method of your choice\n   https://golangci-lint.run/usage/install/#local-installation)
 endif
+
+.PHONY: snapshot
+snapshot: check-goreleaser-install
+	@goreleaser release --snapshot --rm-dist
+
+.PHONY: check-goreleaser-install
+check-goreleaser-install:
+ifeq ("$(shell command -v goreleaser)", "")
+	$(error goreleaser was not found.  Please install it using the method of your choice\n   https://goreleaser.com/install/)
```

Comment:
```shell
make snapshot
Makefile:156: *** goreleaser was not found.  Please install it using the method of your choice\n   https://goreleaser.com/install/.  Stop.
```

Output on Monterrey w/o goreleaser installed.

Reply:
Updated the formatting of these messages for clarity and consistency.